src/js/actions/transform.js

```diff
@@ -209,12 +212,13 @@ define(function (require, exports) {
      *
      * @param {Document} document
      * @param {Layer} targetLayer
-     * @param {{w: number, h: number}} size
+     * @param {{w=: number, h=: number}} size
```

Comment:
Just a tip that you can use properties param in the future to better describe object param:

```
@param {object} size
@param {number=} size.w
@param {number=} size.h
```